tests/page-objects/reports/reports.wdio.page.js

```diff
@@ -110,9 +115,77 @@ const reportsListDetails = async () => {
   return reportDetails;
 };
 
+const collapseSelection = async () => {
+  await (await itemSummary()).click();
+  expect(await (await reportBodyDetails()).isExisting()).to.be.false;
+};
+
+const deleteSelectedReports = async (savedUuids) => {
+  const deleteAllButton = () => $('.action-container .detail-actions .delete-all');
+  const confirmButton = () => $('.btn.submit.btn-danger');
+
+  await (await deleteAllButton()).click();
+  await (await confirmButton()).click();
+  await (await firstReport ()).waitForDisplayed();
+  // make sure the reports are deleted
+  expect(await reportsByUUID(savedUuids[1]).length).to.equal(1);
+};
+
+const deselectAll = async () => {
+  const deselectAllButton = await $('.action-container .deselect-all');
+  await deselectAllButton.click();
+  const count = await $('#reports-content .selection-count > span');
+  expect(await count.isExisting()).to.be.false;
+  expect(await $$(reportBody).length).to.equal(0);
+};
+
+const expandSelection = async () => {
+  await (await itemSummary()).click();
+  await (await $(reportBodyDetailsSelector)).waitForDisplayed();
+};
+const selectAll = async () => {
+  await (await $('.action-container .select-all')).click();
+  await (await $('#reports-content .selection-count > span')).waitForDisplayed();
+  expect(await $$(reportBody).length).to.equal(3);
+};
+
+const selectSeveralReports = async (savedUuids) => {
+  const firstCheck = (await reportsByUUID(savedUuids[0]))[0].$(checkCss);
+  await firstCheck.click();
+  const secondCheck = (await reportsByUUID(savedUuids[2]))[0].$(checkCss);
+  secondCheck.click();
+  await browser.pause(1000);
+  expect(await (await $$(reportBody)).length).to.equal(2);
+};
+
+const selectReport = async (savedUuids) => {
+  const checkbox = await $(`#reports-list li[data-record-id="${savedUuids[0]}"] input[type="checkbox"]`);
+  await checkbox.click();
+  await (await $('#reports-content .selection-count > span:first-child')).waitForDisplayed();
+  expect(await $$(reportBody).length).to.equal(1);
+
+  const textContent = await $('#reports-content .report-body .item-summary .sender .name');
+  await browser.waitUntil(async () => (await textContent.getText()).trim() === 'Sharon');
```

Comment:
I don't think we should use `Sharon` here as a magic string, we should pass it as a parameter.